src/main/java/org/jenkinsci/plugins/build_token_root/BuildRootAction.java

```diff
@@ -188,6 +180,14 @@ private void ok(StaplerResponse rsp) throws IOException {
         }
     }
 
+    private void handleScheduleResult(Queue.Item item, String job, StaplerRequest req, StaplerResponse rsp) throws HttpResponses.HttpResponseException, IOException {
```

Comment:
Does not really help, since you change the signature in #83 anyway.

Reply:
The goal was to change the code in smaller increments, facilitating the bisecting process and keeping refactoring changes from actual changes separate in the history. But I’m fine landing it all in #83 if that’s preferred.